src/data/machines.js

```diff
@@ -23,6 +26,9 @@ export const machineTypes = [
   { name: 'n1-highcpu-96', cpu: 96, memory: 86.4 }
 ]
 
+// returns true if current machine type is smaller than candidate
+export const isMachineTypeSmaller = (current, candidate) => _.find({ name: current }, machineTypes).cpu < _.find({ name: candidate }, machineTypes).cpu
```

Comment:
Do these variable names make sense in the context of this function? It seems like maybe they're coming from the caller. `candidate` seems more fitting for the machine type being tested.

```suggestion
export const isMachineTypeSmaller = (machineTypeA, machineTypeB) => _.find({ name: machineTypeA }, machineTypes).cpu < _.find({ name: machineTypeB }, machineTypes).cpu
```

Reply:
I think its confusing regardless of the names, and included a comment. I'll cchange the names as suggested here though, don't feel too strongly